src/packml-model.js

```diff
@@ -4,7 +4,7 @@
 const JavascripStateMachine = require('javascript-state-machine')
 
 // Define initial start state here
-const INITIAL_START_STATE = 'clearing'
+const INITIAL_START_STATE = process.env.PACKML_INITIAL_START_STATE || 'clearing'
```

Comment:
Can we convert to lowercase? I can see potential misuse by reading the states from the diagram and entering "Execute"